integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -436,17 +436,17 @@ case class LoadTable(
       val columinar = sqlContext.getConf("carbon.is.columnar.storage", "true").toBoolean
 
       // TODO It will be removed after kettle is removed.
-      val useKettle = options.get("useKettle") match {
-        case Some(value) => value.toBoolean
-        case _ =>
-          val useKettleLocal = System.getProperty("use.kettle")
-          if (useKettleLocal == null) {
-            sqlContext.sparkContext.getConf.get("use_kettle_default", "true").toBoolean
-          } else {
-            useKettleLocal.toBoolean
-          }
-      }
-
+//      val useKettle = options.get("useKettle") match {
+//        case Some(value) => value.toBoolean
+//        case _ =>
+//          val useKettleLocal = System.getProperty("use.kettle")
+//          if (useKettleLocal == null) {
+//            sqlContext.sparkContext.getConf.get("use_kettle_default", "true").toBoolean
+//          } else {
+//            useKettleLocal.toBoolean
+//          }
+//      }
```

Comment:
hi, @ravindra, the comment is for verify all test case when use_kettle=false.
and i found that, some commits introduced issues, due to lack of consideration of the new flow